index.html

```diff
@@ -128,101 +128,51 @@ <h3>Multi-Post Stories</h3>
-            <li class="item3"><a href="#">Ruby</a></li>
-          </ul>
-          <button type="button" class="job-button">See Project</button>
-        </div>
-        <div class="job4">
-          <h2>Profesional Art Printing Data</h2>
-          <div class="info-data">
-            <p>
-              A daily section of daily private personalised reads no accounts or
-              sign ups required ha been the industry standups
-            </p>
-          </div>
-          <ul class="grid-container2">
-            <li class="item1"><a href="#">html</a></li>
-            <li class="item2"><a href="#">bootstrap</a></li>
-            <li class="item3"><a href="#">Ruby</a></li>
-          </ul>
-          <button type="button" class="job-button">See Project</button>
-        </div>
-        <div class="job5">
-          <h2>Profesional Art Printing Data</h2>
-          <div class="info-data">
-            <p>
-              A daily section of daily private personalised reads; no accounts
-              or sign ups required ha been the industry standups
-            </p>
-          </div>
-          <ul class="grid-container2">
-            <li class="item1"><a href="#">html</a></li>
-            <li class="item2"><a href="#">bootstrap</a></li>
-            <li class="item3"><a href="#">Ruby</a></li>
-          </ul>
-          <button type="button" class="job-button">See Project</button>
-        </div>
-        <div class="job6">
-          <h2>Profesional Art Printing Data</h2>
-          <div class="info-data">
-            <p>
-              A daily section of daily private personalised reads; no accounts
-              or sign ups required ha been the industry standups
-            </p>
-          </div>
-          <ul class="grid-container2">
-            <li class="item1"><a href="#">html</a></li>
-            <li class="item2"><a href="#">bootstrap</a></li>
-            <li class="item3"><a href="#">Ruby</a></li>
-          </ul>
-          <button type="button" class="job-button">See Project</button>
-        </div>
+      <!--modal-->
+      <div class="modal">
+        <div class="poster poster-modal">
+         <div class="post-information modal-info">
+         <div class="post-title modal-posttitle">
+          <h3 class="heading-modal">Multi-Post Stories</h3>
+          <div class="close-modal">&times;</div>
+         </div>
+         <ul class="grid-container" id="modal-list-grid">
+         
+         </ul>
+         <div class="image-holder image-modal">
+          <img class="post-images" src="./assets/post2.svg" alt="Post" />
+         </div>
+        </div>   
+     <div class="post post-modal">
+      <p class="paragraph"> 
+        A daily selection of private personalised reads; o accounts or
+            signups required. Has been the industry's dummy standard text ever
+            snce the 1500s. When an unknown printer took a standard dummy text
+      </p>
+    </div>
```

Comment:
- [ ] As suggested by the previous reviewer follow the spacing as per the design. Like: 
- [ ] Make sure to give a `margin to the left of the entire popup` as per the design for the mobile version and adjust it to the centre. Also, increase the `project title font size` in the mobile popup.
- [ ] Instead of the 2nd `"See project"`, kindly add `See Source` as per the Figma design.
- [ ] Kindly place the `title` and `close (X) button` in `desktop popup` as per the Figma design.
- Your Mobile version popup:
<img width="305" alt="Screenshot 2022-04-07 at 6 39 50 AM" src="https://user-images.githubusercontent.com/70269507/162116566-19f0b864-88f1-4090-9aa9-99731f94489d.png">
- Figma mobile version popup:
<img width="343" alt="Screenshot 2022-04-07 at 1 45 45 AM" src="https://user-images.githubusercontent.com/70269507/162116884-8e5cda31-d1ac-4379-bb1d-f266c003e8e9.png">

- Your desktop version popup:
<img width="1029" alt="Screenshot 2022-04-07 at 6 40 29 AM" src="https://user-images.githubusercontent.com/70269507/162116575-10bd7c22-4c68-4551-a3a9-1df37c1de6c3.png">
- Figma desktop version popup:
<img width="666" alt="Screenshot 2022-04-07 at 1 47 23 AM" src="https://user-images.githubusercontent.com/70269507/162116870-4bef251e-0ccf-4fa2-b16c-b6ff68c34a48.png">